### ISSUE TITLE
Fix facebook security warning

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#744]
When people sign in or sign up using facebook from https://app.accesslocator.com or https://demo.accesslocator.com, they see some warnings like this:
![image](https://user-images.githubusercontent.com/11838191/50560449-e1a51700-0cce-11e9-8887-20294385dd7f.png)

This appears to require some updated settings in our facebook authentication API.  These settings depend on completion of #742 and #731 (already done).

Facebook authentication was a problem before with issues such as #537 but the underlying problem this time seems to be different.

**Comment by joshi1983** [comment]
I'm going to resolve this by removing both facebook and google buttons.  Both facebook and google require an application process be completed and some manual review.  We want the application maintained without any broken features like this until things are approved.  We'll remove the buttons for now and add them back only after each third party approves us to use them.

**Comment by joshi1983** [comment]
I'm also renaming "Account Recovery" to "Forgotten Password" since no accounts will work through google and facebook authentication for now and "Forgotten Password" is more common.
